.github/workflows/common-config.yaml

```diff
@@ -0,0 +1,52 @@
+# This file is synced with beam-community/common-config. Any changes will be overwritten.
+
+name: Common Config
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/common-config.yaml
+  repository_dispatch:
+    types:
+      - common-config
+  schedule:
+    - cron: "8 12 8 * *"
+  workflow_dispatch: {}
+
+concurrency:
+  group: Common Config
+
+jobs:
+  Sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          persist-credentials: true
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .tool-versions
+
+      - name: Setup Elixir
+        uses: stordco/actions-elixir/setup@v1
+        with:
+          github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```

Comment:
This doesn't specify the elixir and erlang versions here. [By default this action will read from the .tool-versions file](https://github.com/stordco/actions-elixir/blob/main/setup/README.md#usage) and this repo uses differ node, elixir and erlang version than what is noted in common-config repo file.

@btkostner is there some documentation we can / should add to the common-config repo for how to override this file so it doesn't get lost with the periodic syncs? Or is the comment at the top of this file a bit of a scare tactic?

Reply:
This is kinda intentional because we might need to load dependencies and changing versions can break that. With that said, it's not a great solution because it also means we don't know what elixir version our common-config scripts will run against.